Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,15 @@
 import React from 'react';
 import './Card.css';
+
+const CardContent = () => (
+  <div className="card-content">
+    <h1>Get started with React</h1>
+    <p>
+      React makes it painless to create interactive UIs. Design simple views for
+      each state in your application.
+    </p>
+    <h2>reactjs.org</h2>
```

Comment:
This should probably still be a link for user experience purposes, but that opinion is subjective.